modules/pubsub/src/main/resources/application.conf

```diff
@@ -3,6 +3,8 @@
     "type": "PubSub"
     "parallelPullCount": 1
     "maxQueueSize": 3000
```

Comment:
I would set that much higher like 10k or maybe even 100k, given that we have the bytes limit

Reply:
going for 10K after the test against 1KB events

Reply:
decided to stay with 3K after observing memory usage